north-star-vision.md

```diff
@@ -0,0 +1,137 @@
+
+   https://github.com/kubernetes/sig-release/issues/1527
+
+   Outcome: Being able to measure and interpret a set of defined metrics about
+   Kubernetes releases to associate actions with those.
+
+1. **Define and implement the release cadence survey (Introspectable)**
+
+   https://github.com/kubernetes/sig-release/issues/1526
+
+   Outcome: A regular survey evaluating the user experience of the current
+   release cadence.
+
+1. **Simplify CVE process for release management (Secure)**
+
+   https://github.com/kubernetes/sig-release/issues/896
+
+   https://github.com/kubernetes/release/issues/1354
+
+   Outcome: A documented and simple process for handling CVE information within
+   Kubernetes releases.
+
+1. **Establish Cluster API as first-class signal for upstream releases
+   (Secure)**
+
+   Outcome: Cluster API provides a CI signal for blocking release test jobs.
+
+1. **Enhance and simplify Kubernetes version markers (Consumable)**
+
+   Outcome: Clear documentation about available version markers as well as their
+   simplified automation.
+
+### Out of Scope
+
+Topic size estimates are roughly: small (1 week or less), medium (2-3 weeks),
+large (4-5 weeks).
+
+1. **Moving deb/rpm package builds to community infrastructure (large)**
+   - _What:_ The reb/rpm packages are built by the Google build admin, we should
+     move that to community infrastructure and build them automatically.
+   - _Why:_ It’s easier for us to fix issues independently that way and we have
+     more control about how the packages are being built.
+1. **Create a releases landing page (medium)**
+   - _What:_ Creating a releases website as a centralized and canonical place
+     for release related information.
+   - _Why:_ Finding consolidation options where information is right now spread
+     across different places. Helping people to find the latest patch release
+     and where to obtain its artifacts.
+1. **Signing of release artifacts (large)**
+   - _What:_ Being able to GPG sign release artifacts, which also includes
+     container images.
+   - _Why:_ This allows us to enhance the chain of trust between users of
+     Kubernetes artifacts.
+
+### Known Risks
+
+1. **We rely on different SIGs for our work**
+
+   We have a need to discuss our enhancements with different SIGs to get all
+   required information and drive the change. This can lead into helpful, but
+   maybe not expected input and delay the deliverable.
+
+1. **Some topics require initial research**
+
+   We're not completely aware of all technical aspects for the changes. This
+   means that there is a risk of delaying because of investing more time in
+   pre-research.
+
+### Requests to Other Teams
+
+1. **SIG Architecture**
+
+   For the formalization of the released platforms and input about the overall
+   supply chain.
+
+1. **SIG Cluster Lifecycle**
+
+   To get input for making Cluster API a first-class signal for upstream releases.
```

Comment:
I think that it would be useful to clarify what would this improve and what we gain from it.